enterprise/internal/batches/types/changeset.go

```diff
@@ -10,10 +10,13 @@ import (
 	"github.com/goware/urlx"
 	"github.com/inconshreveable/log15"
 	"github.com/sourcegraph/go-diff/diff"
+<<<<<<< HEAD
+=======
 	"github.com/sourcegraph/sourcegraph/internal/gitserver/protocol"
 
 	gerritbatches "github.com/sourcegraph/sourcegraph/enterprise/internal/batches/sources/gerrit"
 	"github.com/sourcegraph/sourcegraph/internal/extsvc/azuredevops"
+>>>>>>> main
```

Comment:
Hey! It seems there are some git markers left over from a rebase operation.
```suggestion
	"github.com/sourcegraph/sourcegraph/internal/gitserver/protocol"

	gerritbatches "github.com/sourcegraph/sourcegraph/enterprise/internal/batches/sources/gerrit"
	"github.com/sourcegraph/sourcegraph/internal/extsvc/azuredevops"
```